projects/varnish/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -eu
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+
+git apply --ignore-whitespace $SRC/patch.diff
+# build project
+./autogen.sh
+./configure
```

Comment:
Why not configure the build with `--enable-asan --enable-ubsan` ?

Reply:
I guess ideally we would fuzz with and without.

Reply:
The sanitizers are set by oss-fuzz through CFLAGS and CXXFLAGS with their preferred options.
Is it ok ?

Reply:
Yes, they are enabled through a sanitizer option in the project YAML.

Can you please add those?

Reply:
These sanitizers are on by default.
And CI looks like it successfully built them, does it not ?